packages/genesys-spark-components/src/components/stable/gux-popover/gux-popover.tsx

```diff
@@ -45,7 +46,7 @@ export class GuxPopover {
    * Indicates the id of the element the popover should anchor to
    */
   @Prop()
-  for: string;
+  for!: string;
```

Comment:
I added this `!` to make the `for` prop required since the popover needs a target element to be clicked to open the popover.